apps/artemis-web/utils/parsers.tsx

```diff
@@ -5,7 +5,9 @@ import { formatDate, genTooltip } from './token';
 
 export function extractUser(req: any): any {
   if (!req.user) return null;
-
+  let type = 'user';
```

Comment:
we also have a `extractLdapUser` function, shouldn't the type also be set there?

Reply:
Nice thought, but extractLdapUser is only being used by the ldap login function, which is defacto about LDAP users :)

Reply:
I get this, but shouldn't we also set the type there? Do we already set it? I.e., if someone connects over LDAP, the `type` var is set or not?

Reply:
Fair point, added that :)

Btw the var is not set in the database, only in the javascript object.

Reply:
Correct, this is carried only on JS side